### PR TITLE
BLEN-264: Basic integration MaterialX nodes into Blender shader nodetree

### PR DESCRIPTION
**Pull request by DagerD**
Partially removed MaterialX node tree and corresponded code.
All MaterialX nodes moved to Blender shader node tree.
MaterialX nodes and Blender nodes now work between each other.

it is possible to have MaterialX `surfacematerial` node or Blender `Material Output` node for material (both works).

Moved functions `import, export, print to console` to Blender shader node tree.

MaterialX library works.
Viewport render works (but not final).

I suggest to do some refactoring and improvements and getting rid of redundant code in separate tasks since here is a lot of testing work already.

### CODE REVIEW

File: materialx/bl_nodes/output.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 # Copyright 2022, AMD
+import MaterialX
```
Comment (bnagirniak) [inline materialx/bl_nodes/output.py]: import MaterialX as mx

File: materialx/bl_nodes/shader.py [hunk 2]
```diff
@@ -2,6 +2,7 @@
 # Copyright 2022, AMD
 
 import math
+import MaterialX
```
Comment (bnagirniak) [inline materialx/bl_nodes/shader.py]: import MaterialX as mx

File: materialx/nodes/node.py [hunk 3]
```diff
@@ -393,17 +427,14 @@ def set_param_value(self, name, value):
 
     @classmethod
     def poll(cls, tree):
-        return tree.bl_idname == utils.with_prefix('MxNodeTree')
+        return tree.bl_idname == bpy.types.ShaderNodeTree.__name__
```
Comment (bnagirniak) [inline materialx/nodes/node.py]: change to 'ShaderNodeTree'

**Review by bnagirniak — CHANGES_REQUESTED** [review]
Combine material/ui.py and nodes.ui.py -> ui.py.
Remove unused code.

**Comment by DagerD** [comment]
There is an issue with draw node in properties area (see picture): the responsible code of drawing it is placed in cycles, eevee, etc modules (look for `panel_node_draw` method), so we are not able to do anything with it on plugin side.
image

File: materialx/material/__init__.py [hunk 4]
```diff
@@ -4,26 +4,11 @@
 import bpy
 
 from . import (
-    ui,
-    properties
+    ui
```
Comment (bnagirniak) [inline materialx/material/__init__.py]: from . import ui

File: materialx/utils.py [hunk 5]
```diff
@@ -472,3 +464,223 @@ def update_materialx_data(depsgraph, materialx_data):
...
+
+
+def get_output_node(material):
+    from .bl_nodes.output import ShaderNodeOutputMaterial
+
+    if not material.node_tree:
+        return None
+
+    bl_output_node = next((node for node in material.node_tree.nodes if
+                 node.bl_idname == ShaderNodeOutputMaterial.__name__ and
```
Comment (bnagirniak) [inline materialx/utils.py]: use 'ShaderNodeOutputMaterial' instead ShaderNodeOutputMaterial.__name__

**Review by bnagirniak — COMMENTED** [review]
small fixes

**Review by bnagirniak — APPROVED** [review]

**Pull request merged**